index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shahier</title>
+    <script src="https://kit.fontawesome.com/a3ad5bee92.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav>
+            <h3>Welcome</h3>
+            <div>
+                <a href="#"><i class="fas fa-bars fa-2x"></i></a>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="head">
+            <h2>Hey there. I’m Shahier</h2>
+            <h2>I’m a software developer</h2>
+            <div>
+                I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </div>
+        </div>
+        <div class="icons">
+            <a href="#"><i class="fab fa-twitter"></i></a>
+            <a href="#"><i class="fab fa-linkedin-in"></i></a>
+            <a href="#"><i class="fab fa-medium-m"></i></a>
+            <a href="#"><i class="fab fa-github"></i></a>
+            <a href="#"><i class="fab fa-angellist"></i></a>
+        </div>
```

Comment:
The list of icons should use a ul tag to group them, and li per icon; each icon has a link (a tag).